packages/form-builder/src/components/canvas/DesignPreview.tsx

```diff
@@ -1,41 +1,53 @@
 import * as React from 'react';
 import {useEffect, useState} from 'react';
-import DesignPreviewLoading from './DesignPreviewLoading';
 
 import Storage from '@givewp/form-builder/common/storage';
 
 import IframeResizer from 'iframe-resizer-react';
 import {useFormState} from '../../stores/form-state';
+import DesignPreviewLoading from "@givewp/form-builder/components/canvas/DesignPreviewLoading";
 
 const DesignPreview = () => {
     const {blocks, settings: formSettings} = useFormState();
-    const [sourceDocument, setSourceDocument] = useState('');
-    const [isLoading, setIsLoading] = useState<boolean>();
+    const [isLoading, setLoading] = useState(true);
+    const [sourceDocument, setSourceDocument] = useState(null);
+    const [previewHTML, setPreviewHTML] = useState(null);
 
```

Comment:
Let's add the generics to these useStates:

```
const [isLoading, setIsLoading] = useState<boolean>();
const [sourceDocument, setSourceDocument] = useState<string>();
```

As a note I prefer setting default loading states to false because that value should always be programmatically set based on data fetching.

Reply:
Ah, yes - I knew I was forgetting something.

Reply:
Resolved by https://github.com/impress-org/givewp-next-gen/pull/91/commits/3565f13cfbe381bda8044374cc50ba030a133e8e.